config/v1/types_apiserver.go

```diff
@@ -39,6 +39,9 @@ type APIServerSpec struct {
 	// The values are regular expressions that correspond to the Golang regular expression language.
 	// +optional
 	AdditionalCORSAllowedOrigins []string `json:"additionalCORSAllowedOrigins,omitempty"`
+	// encryption allows the configuration of encryption types for encryption of resources at the datastore layer.
+	// +optional
+	Encryption APIServerEncryption `json:"encryption,omitempty"`
```

Comment:
this must be a pointer

Reply:
Why?  Unset is handled correctly.

Reply:
omitempty does not work for non-pointers. Choose one.